### ISSUE TITLE
[Feature Request] Budget based on Pay days

### ISSUE THREAD

**Issue by dudefoxlive** [ellite/Wallos#386]
I would like to request a feature where I can add the days that I get paid and have it tell me how much money I need to set aside per pay period. For example I get paid on the 12th and my next paycheck isn't until the 26th. It would be nice to know how much I have to set aside per pay period.

**Comment by morenicano** [comment]
YES! All of this! My wife and I use a spreadsheet and list the bills according to pay period. It'd be nice to be able to add a pay calendar (every x weeks/months/year[for things like bonuses or tax refunds] specific days of the month), and then allow the subscription calendar to show the pay periods and what subscriptions are due during those time periods.

**Comment by DamagedDingo** [comment]
I was considering requesting something similar. For example, I get paid fortnightly (Wednesday), and it would be incredibly helpful to have the following features:

**Payments Due Before the Next Pay Date:** A view that shows all payments due before the next pay date (currently I'm using the calendar, but I'm new so hopefully am not missing something more obvious).

**Payment Totals and Breakdown:** A summary of the total amount for all payments due within a given two-week period. Additionally, a breakdown by payment location would help ensure funds are correctly allocated.

**Adjusting Payment Dates:** Off-topic, but in cases where money is tight, it would be great to temporarily skip a payment by moving its date without affecting future payment schedules. For example I needed to move my Phone bill 2 weeks later over Christmas.
